Neo4j.Driver/Neo4j.Driver/V1/ISession.cs

```diff
@@ -47,16 +47,10 @@ public interface ISession : IStatementRunner
         /// method.
         ///
         /// </summary>
+        /// <param name="txConfig">Configuration for the new transaction.
+        /// If not specified, <see cref="TransactionConfig.Empty"/> will be used.</param>
         /// <returns>A new transaction.</returns>
-        ITransaction BeginTransaction();
-
-        /// <summary>
-        /// This method is deprecated.
-        /// </summary>
-        /// <param name="bookmark">A bookmark.</param>
-        /// <returns>A new transaction.</returns>
-        [Obsolete("Pass in bookmark at session instead.")]
-        ITransaction BeginTransaction(string bookmark);
+        ITransaction BeginTransaction(TransactionConfig txConfig=null);
```

Comment:
Shall we extract method overloads instead of default parameters (for better C++/CLI integration) in the rest of the interface?